### PR TITLE
DPAD-34 | Add heartbeat tracking

### PR DESCRIPTION
**Pull request by JoshuaRogan**
* Fires events for ads at one second intervals for first ten seconds then one per ten seconds
* Fires event for content at 10 seconds intervals until the first minute than at one minute intervals

### CODE REVIEW

File: src/utils/videoTracking.ts [hunk 1]
```diff
@@ -92,6 +94,8 @@ function addRunTimeParams(trackingParams: TrackData): TrackData {
 	trackDataObject[PROPERTY_NAMES.VIDEO_DURATION] = getDuration();
 	trackDataObject[PROPERTY_NAMES.VIDEO_PLAYER] = getPlayerId();
 	trackDataObject[PROPERTY_NAMES.VIDEO_VIEW_STATE] = getJWViewState();
+	trackDataObject[PROPERTY_NAMES.VIDEO_IS_EMBEDDED] = getIsEmbed();
+	trackDataObject[PROPERTY_NAMES.VIDEO_QUALITY_MANIFEST] = getJWQualityManifest();
```
Comment (p-forfa) [inline src/utils/videoTracking.ts:98]: So are we storing the available quality levels for the player, whenever it loads?

Comment (JoshuaRogan) [reply src/utils/videoTracking.ts:98]: This is the _possible_ qualities that the video can player. It's available as soon as the player loads. Which all of these events fire after

Comment (p-forfa) [reply src/utils/videoTracking.ts:98]: Just saw it in the doc - https://docs.google.com/spreadsheets/d/1zBUIt8fxmjEItf9HiW9TU1EZnopSLju_TO-BEEM-WuE/edit#gid=1564524057

If I understand it correctly, we'll be sending these the player quality values on each tracking event. The thing I'm uncertain about is whether we'll also send the current quality that the player is watching the video at, not just all the available ones.

**Review by p-forfa — APPROVED** [review]

File: src/utils/videoTracking.ts (hunk 1 above)
Comment (JoshuaRogan) [reply src/utils/videoTracking.ts:98]: Yup, already done via `video_current_quality`

**Pull request merged**